modular_skyrat/modules/loadouts/loadout_items/_loadout_datum.dm

```diff
@@ -38,8 +38,8 @@ GLOBAL_LIST_EMPTY(all_loadout_datums)
 /datum/loadout_item
 	/// Displayed name of the loadout item.
 	var/name
-	/// Whether this item can be renamed.
-	var/can_be_named = FALSE
+	/// Whether this item can be renamed and described.
+	var/can_be_named = TRUE
```

Comment:
Do we have anything with this set to FALSE? _Should_ we have anything that sets it to FALSE? Anything we should be concerned about for this?

Reply:
only thing that used this var was plushies before